app/views/main.scala.html

```diff
@@ -178,13 +171,15 @@ <h3>Social networks</h3>
             </div>
             <div class="lb-legal">
                 <p>
-                    &copy; Lightbend 2021 |
                     <a href="https://www.lightbend.com/legal/licenses" target="_blank">Licenses</a> |
                     <a href="https://www.lightbend.com/legal/terms" target="_blank">Terms</a> |
                     <a href="https://www.lightbend.com/legal/privacy" target="_blank">Privacy Policy</a> |
                     <a href="@routes.Application.cookie">Cookie Listing</a> |
                     <a class="optanon-toggle-display">Cookie Settings</a>
                 </p>
+                <p>
+                    &copy; 2011-2021 Lightbend Inc., since 2022 the original author or authors from the Play Framework project
+                </p>
```

Comment:
That's the copyright we came up with in the slack channel a couple of weeks ago.

Reply:
I use 2011 instead of 2009, because that's when Typesafe came in. Also the first commit to Play 2 was in Sept. 2011